streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -133,6 +133,24 @@ public void close(final boolean clean) {
         }
     }
 
+    @Override
+    public void closeSuspended(final boolean clean, final RuntimeException e) { }
```

Comment:
Ditto as below, for these four functions could we be more strict and not calling it in `StreamThread` and throw unsupported exception if ever called?